OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -179,7 +186,27 @@ public void Tick(Actor self)
 		public virtual IEnumerable<IRenderable> Render(Actor self, WorldRenderer wr)
 		{
 			return world.ScreenMap.FrozenActorsInBox(owner, wr.Viewport.TopLeft, wr.Viewport.BottomRight)
-				.Where(f => f.Visible)
+				.Where(f =>
```

Comment:
OK, I unnestified the if statements. I don't think it can be simplified more than this.